src/plugins/charts/public/services/colors/color_palette.ts

```diff
@@ -65,8 +65,8 @@ export function createColorPalette(num: number): string[] {
     throw new TypeError('ColorPaletteUtilService expects a number');
   }
 
-  const colors = seedColors;
-  const seedLength = seedColors.length;
+  const colors = euiPaletteColorBlind();
+  const seedLength = euiPaletteColorBlind().length;
```

Comment:
nit - could just use 
```suggestion
  const seedLength = colors.length;
```